Small tweak in src\Wasmtime.csproj can remove dependency on "unzip"

I am using Windows and do not have 'unzip' in my path (I use 7-zip)

Replacing the MSBuild `Exec` task with another built-in MSBuild  task `Unzip` removes the need to have

Trying to build

`PS> cd src`
`PS> dotnet build .\Wasmtime.sln`

results in this error

```
E:\MyGithub\wasmtime-dotnet\src\Wasmtime.csproj(166,5): error MSB3073: The command "unzip wasmtime-dev-x86_64-windows-c-api.zip" exited with code 900
9.
C:\Program Files\dotnet\sdk\7.0.305\Microsoft.Common.CurrentVersion.targets(5150,5): error MSB3030: Could not copy the file "E:\MyGithub\wasmtime-dot
net\src\obj\wasmtime-dev-x86_64-windows-c-api\lib\wasmtime.dll" because it was not found. [E:\MyGithub\wasmtime-dotnet\src\Wasmtime.csproj::TargetFra
mework=net7.0]
```

I'll be happy to contribute a PR with the switch if you are interested.

--- a/src/Wasmtime.csproj
+++ b/src/Wasmtime.csproj
@@ -167,8 +167,6 @@ The .NET embedding of Wasmtime enables .NET code to instantiate WebAssembly modu
       Condition="('$(Packing)'=='true' Or '%(Wasmtime.Copy)'=='true') And %(Wasmtime.FileExtension) == 'zip' And !Exists('$(BaseIntermediateOutputPath)$(ReleaseFileNameBase)-%(Wasmtime.Arch)-%(Wasmtime.OS)-c-api')"
       SourceFiles="$(BaseIntermediateOutputPath)\$(ReleaseFileNameBase)-%(Wasmtime.Arch)-%(Wasmtime.OS)-c-api.%(Wasmtime.FileExtension)"
       DestinationFolder="$(BaseIntermediateOutputPath)"
-      StandardOutputImportance="Low"
-      StandardErrorImportance="Low"
     />
 
     <!-- The content here is specifically for packing -->

--- a/src/Wasmtime.csproj
+++ b/src/Wasmtime.csproj
@@ -163,10 +163,10 @@ The .NET embedding of Wasmtime enables .NET code to instantiate WebAssembly modu
       StandardOutputImportance="Low"
       StandardErrorImportance="Low"
     />
-    <Exec
+    <Unzip
       Condition="('$(Packing)'=='true' Or '%(Wasmtime.Copy)'=='true') And %(Wasmtime.FileExtension) == 'zip' And !Exists('$(BaseIntermediateOutputPath)$(ReleaseFileNameBase)-%(Wasmtime.Arch)-%(Wasmtime.OS)-c-api')"
-      Command="unzip $(ReleaseFileNameBase)-%(Wasmtime.Arch)-%(Wasmtime.OS)-c-api.%(Wasmtime.FileExtension)"
-      WorkingDirectory="$(BaseIntermediateOutputPath)"
+      SourceFiles="$(BaseIntermediateOutputPath)\$(ReleaseFileNameBase)-%(Wasmtime.Arch)-%(Wasmtime.OS)-c-api.%(Wasmtime.FileExtension)"
+      DestinationFolder="$(BaseIntermediateOutputPath)"
       StandardOutputImportance="Low"
       StandardErrorImportance="Low"
     />

